recipe/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "forex-python" %}
+{% set version = "1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MicroPyramid/forex-python/archive/v{{ version }}.tar.gz
```

Comment:
By using this source the version reported by pip will be 1.6, see https://github.com/MicroPyramid/forex-python/blob/v1.8/setup.py#L5. WHich tells me that the source in GH potentially doesn't match what's on PyPI.

Reply:
I think it would be appropriate to patch that `setup.py` file.

Reply:
I would actually diff the source files to see if there is any difference.

Reply:
The tags `1.6` and `v1.8` are few commits apart in the history so they have to differ. Weird that `v1.7` and `v1.8` point to the same commit. In any case, whatever the upstream developer considers 1.8, we can only assume is what we have to pack.

Reply:
I don't trust upstream in this case. If GH and PyPI differ, we need to look at the diff and decide what to do.

Reply:
I thought PyPi just redistributes whatever comes from GH. Maybe still worth to check though.

Reply:
PyPI has no notion of what GH is and nothin stops a maintainer from releasing any code from anywhere.

Reply:
PyPi links PR, issues, forks, etc. from the GH project so it should have notion of it, but ok, I get that the releases may still not come from GH.

Reply:
>PyPi links PR, issues, forks, etc

People add links via package metadata. PyPI only adds icons to links to links provided in package metadata.

Reply:
So your concern is that someone may have uploaded different source code that GH, albeit the links that come from the package metadata point to the GH project. It sounds like a really weird scenario, but I do agree it's worth to check for differences.

Reply:
It doesn't happen often, but it does happen.

Reply:
I continue to think that we should take the source code from PyPI.